### DISCUSSION TITLE
View source within the CKEditor5 in Visual Editor

### DISCUSSION THREAD

**Discussion by arussell-wc** [requarks/wiki#6754] [Help / Questions] [answered]
I know i can probably modify the properties/configuration within the CKEditor that is embedded in the source code. But is there some way within Wiki.js to enable the editing of the source code for a page within the WYSIWYG editor? 

https://ckeditor.com/docs/ckeditor5/latest/features/source-editing.html

**Comment by NGPixel** [comment, marked as answer]
No, the ckeditor used is a much older version which didn't support source code editing at the time. You would need to upgrade the dependency and fix any breaking changes.

**Reply by arussell-wc** [reply]
Excellent! Thank you so much for the prompt response, i will look into this and submit a PR if thats okay with you. Hopefully by the end of October, as this is one of many projects im working on for the company im with.

**Comment by 2b0x** [comment]
Hi, i have a some questions, i want to add a customize button in toolbar, but i cant find a way to add it. Have you solved this problem?
